lib/auth.js

```diff
@@ -33,10 +34,10 @@ function createOAuthToken (username, password, secondFactor) {
 function saveToken ({email, token}) {
   const Netrc = require('netrc-parser')
   const netrc = new Netrc()
-  const hosts = ['api.heroku.com', 'git.heroku.com']
+  const hosts = [vars.apiHost, vars.gitHost]
   hosts.forEach(host => {
-    netrc.machines[host].login = email
-    netrc.machines[host].password = token
+    netrc.machines[vars.apiHost].login = email
```

Comment:
Now you're always setting the `vars.apiHost` and not the actual `host` which will break all Git clients...